### ISSUE TITLE
Is it possible to make footnotes appear somewhere other than the bottom of the document?

### ISSUE THREAD

**Issue by asmeurer** [executablebooks/MyST-Parser#179]
With RST you can use `rubric: footnotes` to make footnotes appear wherever you want. Is it possible to do this in MyST? It seems to only put them at the bottom of the document.

**Comment by asmeurer** [comment]
Also would https://github.com/executablebooks/MyST-Parser/issues/164 make "rubric: footenotes" do what I want, or would MyST still try to put the footnotes at the bottom?

**Comment by chrisjsewell** [comment]
No, not at present, all footnotes are gathered during parsing, then output at the end of the document in https://github.com/executablebooks/MyST-Parser/blob/877b80cd64cd254abe965b4cfb6b0c7abe2b29ab/myst_parser/docutils_renderer.py#L67

**Comment by chrisjsewell** [comment]
Do you have an example of the kind of outcome you are looking to achieve?

**Comment by asmeurer** [comment]
https://www.asmeurer.com/brown-water-python/tokens.html#op

I generate the table as a rst list table https://github.com/asmeurer/brown-water-python/blob/master/docs/exact_type_table.py. It would be possible, though a little more annoying, to do this with a Markdown table, but I would prefer to keep the footnotes next to the table as they are now.

**Comment by chrisjsewell** [comment]
Yep sounds reasonable; I have done something similar with tables in latex.
Ideally what I think you would want here is a "scoped" footnote section, i.e. at the bottom of your table, you only want footnotes from that table, not footnotes before or after the table

**Comment by adamgayoso** [comment]
+1 for this feature!
